TestsDts/u-boot-env-size-40000.dts

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "librerouter,librerouter-r2", "mediatek,mt7621-soc";
+	model = "LibreRouter R2";
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status {
+			label = "yellow:status";
+			color = <LED_COLOR_ID_YELLOW>;
+			function = LED_FUNCTION_STATUS;
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <44000000>;
+		broken-flash-reset;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "u-boot";
+				reg = <0x0 0x30000>; 
+				read-only;
+			};
+
+			partition@30000 {
+				label = "u-boot-env";
+				reg = <0x30000 0x40000>; 
```

Comment:
This change is for probe changing the size of partition. Im following the common data what set this size for the env partition.

Responses:
In the fw
![Captura de pantalla de 2024-10-21 17-12-06](https://github.com/user-attachments/assets/36bea1ba-c787-4f33-be9c-32f3a8eaac59)

In the uboot console
![imagen](https://github.com/user-attachments/assets/b8e8dcc3-4c8e-458d-8b15-a1160999e5b0)

Observations: The u-boot-env had save old variables

Reply:
interesante para leer
https://docs.u-boot.org/en/latest/api/bootcount.html
https://github.com/u-boot/u-boot/blob/master/doc/api/bootcount.rst 
https://community.toradex.com/t/compilation-of-u-boot-with-config-bootcount-limit/5807 

para agregar el config ... 
https://git.openwrt.org/?p=openwrt/openwrt.git;a=blob;f=package/boot/uboot-envtools/files/mvebu;h=7d16a59666b0b145f9b7f1e4e80fbc9cf8eb7699;hb=bfbdeeb3de3da31f7e5f9bd429e079c2d839644a

mirar este pull para inspirarse ... https://github.com/openwrt/openwrt/pull/4236/files#diff-83f72b6c9bf6ff6743765af322397007c65353f1ace798e141dc6ef5dda5cdff

Reply:
basicamente estaba toda la info necesaria aqui https://openwrt.org/docs/techref/bootloader/uboot.config pero no lo leimos detenidamente 
Anda agregando toda la info que vemos que sirve en el reporte